ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -45,6 +45,7 @@ component:
   pump: "Component used for the distribution of liquid media."
   riser: "Water riser for heating, cooling and so on"
   shade: "Window covering."
+  signal: "An electric current or electromagnetic field used to convey data from one place to another."
```

Comment:
is this really a `component`? this seems like a `descriptor` to me.

Reply:
ack this should be a descriptor